### PR TITLE
:bug: use mime-type application/ld+json instead of text/plain

### LINKED ISSUE #28: Output cannot be automatically parsed as json/jsonld
#### Issue type:

- :bug: Bug 

____

#### Short description:

When fetching from the api the content-type is `text/plain`

#### Steps to reproduce the problem:

1. `curl -i URL_TO_API`
2. check `content-type`

#### Expected behaviour:

The `content-type` is application/ld+json
____

#### Versions:

- This mod: all
- Minecraft: all
- Forge: all

### PR DESCRIPTION
**Pull request by thunfisch987**
closes #28

### CODE REVIEW

**Comment by thunfisch987** [comment]
~i did not manage to build it on my system, is there a way to do that?
(i don't have much experience with minecraft modding)
i want to try including an option for https via a config option but i would need to build it on my system for testing~

i got it working (only thing missing was my github username and token)

**Comment by thunfisch987** [comment]
Since i got it to build on my machine i tested it and it works!

File: src/main/java/org/cyclops/integratedrest/http/HttpServerHandler.java [hunk 1]
```diff
@@ -100,7 +100,7 @@ private boolean writeResponse(HttpRequest request, ChannelHandlerContext context
                 HttpVersion.HTTP_1_1, responseStatus,
                 Unpooled.copiedBuffer(responseString, CharsetUtil.UTF_8));
 
-        response.headers().set(HttpHeaderNames.CONTENT_TYPE, "text/plain; charset=UTF-8");
+        response.headers().set(HttpHeaderNames.CONTENT_TYPE, "application/ld+json; charset=UTF-8");
```
Comment (rubensworks) [inline src/main/java/org/cyclops/integratedrest/http/HttpServerHandler.java]:
```suggestion
        response.headers().set(HttpHeaderNames.CONTENT_TYPE, "application/ld+json");
```

**Comment by rubensworks** [comment]
Thanks @thunfisch987!
Will prepare a new release in a bit.

File: src/main/java/org/cyclops/integratedrest/http/HttpServerHandler.java (hunk 1 above)
Comment (thunfisch987) [reply src/main/java/org/cyclops/integratedrest/http/HttpServerHandler.java]: Afaik the charset is actually redundant so it could definitely be removed

**Pull request merged**